specification/communitytraining/resource-manager/Microsoft.Community/stable/2023-11-01/CommunityTrainings.json

```diff
@@ -60,7 +60,7 @@
         ],
         "responses": {
           "200": {
-            "description": "ARM operation completed successfully.",
+            "description": "Azure operation completed successfully.",
```

Comment:
I am just curious why this change happened. Is it because the new TypeSpec includes this?
- https://github.com/Azure/typespec-azure/pull/176

Reply:
Ah, yeS:

https://github.com/Azure/typespec-azure/pull/176/files#diff-790dc31a8ce452e4359c87535953eba36008664ceef11d0c3ecd0f167626fcf8